pkg/cli/admin/nodeimage/create_test.go

```diff
@@ -133,6 +133,14 @@ func TestRun(t *testing.T) {
 		}
 	}
 
+	ClusterVersion_4_16_ObjectFn := func(repo string, manifestDigest string) []runtime.Object {
```

Comment:
Unit tests are failing because defaultClusterVersionObjectFn's Status.Desired.Version needs to be set. The same should be applied to monitor_test.go.

Reply:
Tests are now failing, added a check in the code